src/Statistics/EmployeeParticipationShare.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+
+namespace VSV\GVQ_API\Statistics;
```

Comment:
Bij de andere domain models is er een ValueObjects subfolder, daar mag deze class in.